lookout/style/format/benchmarks/evaluate_smoke.py

```diff
@@ -153,7 +153,7 @@ def calc_aligned_metrics(bad_style_code: str, correct_style_code: str, generated
     return misdetection, undetected, detected_wrong_fix, detected_correct_fix
 
 
-def calc_metrics(bad_style_code: str, correct_style_code: str, fe: Optional[FeatureExtractor],
+def  calc_metrics(bad_style_code: str, correct_style_code: str, fe: Optional[FeatureExtractor],
```

Comment:
@zurk Why is everything optional here? I prefer to force existing values everywhere unless we really need otherwise.

Reply:
I agree, but it is not so easy. because you can have 0 fixes for some files. 
I keep a todo for the future.

Reply:
If you have zero fixes, you still pass a valid FeatureExtractor instead of None, don't you?

Reply:
no, I just do not have anything in this case. because I take `FeatureExtractor` from `FixData` and if there is no `FixData` => no `FeatureExtractor`